ansible_collection/hpe/nimble/plugins/modules/hpe_nimble_volume_collection.py

```diff
@@ -0,0 +1,722 @@
+        if utils.is_null_or_empty(volcoll_resp):
+            return (False, False, f"Volume collection '{volcoll_name}' not present to demote.", {})
+        else:
+            client_obj.volume_collections.demote(id=volcoll_resp.attrs.get("id"), **params)
+            return (True, True, f"Demoted volume collection '{volcoll_name}' successfully.", {})
+    except Exception as ex:
+        return (False, False, f"Demote volume collection failed | {ex}", {})
+
+
+def handover_volcoll(
+        client_obj,
+        volcoll_name,
+        **kwargs):
+
+    if utils.is_null_or_empty(volcoll_name):
+        return (False, False, "Handover of volume collection failed as volume collection name is null.", {})
+
+    try:
+        volcoll_resp = client_obj.volume_collections.get(id=None, name=volcoll_name)
+        params = utils.remove_null_args(**kwargs)
+        if utils.is_null_or_empty(volcoll_resp):
+            return (False, False, f"Volume collection '{volcoll_name}' not present for handover.", {})
+        else:
+            client_obj.volume_collections.handover(id=volcoll_resp.attrs.get("id"), **params)
+            return (True, True, f"Handover of volume collection '{volcoll_name}' done successfully.", {})
+    except Exception as ex:
+        return (False, False, f"Handover of volume collection failed | {ex}", {})
+
+
+def abort_handover_volcoll(
+        client_obj,
+        volcoll_name):
+
+    if utils.is_null_or_empty(volcoll_name):
+        return (False, False, "Abort handover of volume collection failed as volume collection name is null.", {})
+
+    try:
+        volcoll_resp = client_obj.volume_collections.get(id=None, name=volcoll_name)
+        if utils.is_null_or_empty(volcoll_resp):
+            return (False, False, f"Volume collection '{volcoll_name}' not present for abort handover.", {})
+        else:
+            client_obj.volume_collections.abort_handover(id=volcoll_resp.attrs.get("id"))
+            return (True, True, f"Abort handover of volume collection '{volcoll_name}' done successfully.", {})
+    except Exception as ex:
+        return (False, False, f"Abort handover of volume collection failed | {ex}", {})
+
+
+def validate_volcoll(
+        client_obj,
+        volcoll_name):
+
+    if utils.is_null_or_empty(volcoll_name):
+        return (False, False, "Validate volume collection failed as volume collection name is null.", {})
+
+    try:
+        volcoll_resp = client_obj.volume_collections.get(id=None, name=volcoll_name)
+        if utils.is_null_or_empty(volcoll_resp):
+            return (False, False, f"Volume collection '{volcoll_name}' not present for validation.", {})
+        else:
+            client_obj.volume_collections.validate(id=volcoll_resp.attrs.get("id"))
+            return (True, False, f"Validation of volume collection '{volcoll_name}' done successfully.", {})
+    except Exception as ex:
+        return (False, False, f"Validation of volume collection failed | {ex}", {})
+
+
```

Comment:
Is it pythonic to have double line breaks between functions?

Reply:
ya. as a good practice they recommend

https://www.flake8rules.com/rules/E302.html